cloudigrade/util/aws.py

```diff
@@ -70,6 +71,12 @@ def is_running(cls, code):
         return code == cls.running.value
 
 
+class InvalidArn(SuspiciousOperation):
```

Comment:
This exception is probably better off living in `exceptions.py`.

Reply:
I only see account/exceptions.py, which doesn't seem right. Are you meaning "create a new exceptions.py file" ?

Reply:
Yup!